### PR TITLE
fix: adds spacing to contributor highlights container

### LINKED ISSUE #1057: bug: add more spaces between highlights
### Type of feature

🍕 Feature

### Current behavior

close-edit

### Suggested solution

Add more space between highlights.
@open-sauced/triage Thoughts?

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by bdougie**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR addresses the spacing of highlights on contributor profiles. They seem to be on top of each other and my change is adding a simple `mt-6` to the div.

I am open to other suggestions, but trying to keep it simple for this. 

## Related Tickets & Documents

fix #1057

## Mobile & Desktop Screenshots/Recordings

Screen Shot 2023-04-11 at 2 46 42 PM

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by takanome-dev — APPROVED** [review]
Looks good :+1:

**Comment by OgDev-01** [comment]
This works... But it will be better to add a `space-y` property to the container instead to avoid weird margins in places we don't need them 🤌

**Comment by bdougie** [comment]
> This works... But it will be better to add a `space-y` property to the container instead to avoid weird margins in places we don't need them 🤌 

Of course. I will update this to use space-y

**Review by brandonroberts — APPROVED** [review]

**Comment by bdougie** [comment]
> This works... But it will be better to add a `space-y` property to the container instead to avoid weird margins in places we don't need them 🤌

space-y only adds margin top. 

https://tailwind.build/classes/spacing/-space-y-1+%25_F+.-space-y-*

Going to switch back to mb-6 and take the tech debt for now.

File: components/organisms/ContributorProfileTab/contributor-profile-tab.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ const ContributorProfileTab = ({
                 <div>
                   {/* eslint-disable-next-line camelcase */}
                   {highlights.map(({ id, title, highlight, url, created_at }) => (
-                    <div className="flex flex-col gap-2 lg:flex-row lg:gap-7" key={id}>
+                    <div className="flex flex-col gap-2 lg:flex-row lg:gap-7 space-y-2" key={id}>
```
Comment (bdougie) [inline components/organisms/ContributorProfileTab/contributor-profile-tab.tsx:124]:
```suggestion
                    <div className="flex flex-col gap-2 lg:flex-row lg:gap-7 mb-6" key={id}>
```

**Pull request merged**